File: php/src/Board.php
<?php

namespace TicTacToe;

class Board
{
    /** @var Tile[] */
    private $plays;

    public function __construct()
    {
        for ($i = 0; $i < 3; $i++) {
            for ($j = 0; $j < 3; $j++) {
                $tile = new Tile();
                $tile->x = $i;
                $tile->y = $j;
                $tile->symbol = ' ';
                $this->plays[] = $tile;
            }
        }
    }

    public function tileAt(int $x, int $y): Tile
    {
        foreach ($this->plays as $t) {
            if ($t->x == $x && $t->y == $y){
                return $t;
            }
        }
        return null;
    }

    public function addTileAt(string $symbol, int $x, int $y): void
    {
        $newTile = new Tile();
        $newTile->x = $x;
        $newTile->y = $y;
        $newTile->symbol = $symbol;

        $this->tileAt($x, $y)->symbol = $symbol;
    }
}

File: python/Board.py
from Tile import Tile


class Board(object):

    def __init__(self):
        self._plays = []
        for i in range(3):
            for j in range(3):
                tile = Tile()
                tile.X = i
                tile.Y = j
                tile.Symbol = ' '
                self._plays.append(tile)

    def AddTileAt(self, symbol, x, y):
        new_tile = Tile()
        new_tile.X = x
        new_tile.Y = y
        new_tile.Symbol = symbol

        self.TileAt(x, y).Symbol = symbol

    def TileAt(self, x, y):
        for t in self._plays:
            if t.X == x and t.Y == y:
                return t
        return None
